Gemfile

```diff
@@ -98,4 +98,4 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'sentry-raven', '~> 2.9'
 
-gem 'et_dropzone_uploader', path: 'vendor/gems/et_dropzone_uploader'
\ No newline at end of file
+gem 'et_dropzone_uploader', git: 'https://github.com/hmcts/et_dropzone_uploader.git'
```

Comment:
One comment - dont forget to lock it to a commit or a tag else it will change every time someone builds it

Reply:
Yup, it's in the README but waiting for that PR before I add the tag :)

Reply:
Ok, go for it when you are ready then - ive approved the gem